log.go

```diff
@@ -84,7 +84,7 @@ func InitLoggerWithWriteSyncer(cfg *Config, output zapcore.WriteSyncer, opts ...
 		return nil, nil, err
 	}
 	core := NewTextCore(encoder, output, level)
-	opts = append(cfg.buildOptions(output), opts...)
+	opts = append(cfg.buildOptions(os.Stderr), opts...)
```

Comment:
I tend to give this choice to user who use this library, like zap: https://github.com/uber-go/zap/blob/master/config.go#L82-L92

Reply:
It's also have some compatibility issues, such as TiDB Dashboard cannot search ERROR logs.

Reply:
By the way, i wonder how this option affects TiDB Dashboard? Do you mean the internal error of zap will also be showed on dashboard?

Reply:
@zhaoxinyu Yes, all error level logs. dashboard's log search result is from the log file.